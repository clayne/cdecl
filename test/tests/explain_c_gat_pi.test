cdecl @ @ @ explain int *__attribute__(( foo )) const p @ 0
