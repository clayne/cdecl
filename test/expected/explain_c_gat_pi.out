explain int *__attribute__(( foo )) const p
             ^
14: warning: "__attribute__" is not supported by cdecl (ignoring); did you mean [[...]]?
declare p as constant pointer to int
